src/css/typography.sass

```diff
@@ -0,0 +1,165 @@
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans&family=Poppins:wght@400;500;600;700&display=swap")
+@import 'colors'
+
+// $thin: 100
+// $extra-light: 200
+// $light: 300
+$regular: 400
+$medium: 500
+$semi-bold: 600
+$bold: 700
+// $extra-bold: 800
+// $black: 900
+// $extra-black: 950
+
+=header-globals()
+  font-family: 'Poppins', sans-serif
+
+=t-h1()
+  +header-globals()
+  font-weight: $bold
+  font-size: 56px
+  line-height: 64px
+  letter-spacing: 0.01em
+
+=t-h2()
+  +header-globals()
+  font-weight: $semi-bold
+  font-size: 40px
+  line-height: 56px
+
+=t-h3()
+  +header-globals()
+  font-weight: $bold
+  font-size: 36px
+  line-height: 40px
+  letter-spacing: 0.01em
+
+=t-h4()
+  +header-globals()
+  font-weight: $semi-bold
+  font-size: 32px
+  line-height: 40px
+
+=t-h5()
+  +header-globals()
+  font-weight: $semi-bold
+  font-size: 24px
+  line-height: 30px
+  letter-spacing: 0.01em
+
+=t-h6()
+  +header-globals()
+  font-weight: $semi-bold
+  font-size: 20px
+  line-height: 24px
+
+=t-subtitle1()
+  +header-globals()
+  font-weight: $semi-bold
+  font-size: 16px
+  line-height: 20px
+
+=t-subtitle2()
+  +header-globals()
+  font-weight: $medium
+  font-size: 14px
+  line-height: 20px
+  letter-spacing: 0.005em
```

Comment:
feel free to delete these other typography classes/mixins that are not used in the design

Reply:
I deleted unused classes. Updating typography.sass has changed the text on the component builder. Text became white, so I've taken the white colour off the sass file and put it in the TypographyExpose.vue styles. The text is the wrong size now on the buttons but I don't want to start fiddling with the component builder.
I'm thinking that the component builder and the components should be more siloed ideally.